entries/n/n-able.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Cove Data Protection": {
```

It looks like you're trying to a product by N-able that shares authentication process with all the other N-able products.
As such I recommend you rename the entry to just N-able. You can then, should you wish, add the different domains of each of the products as additional-domains.
```suggestion
  "N-able": {
```